README.md

```diff
@@ -79,8 +79,17 @@ The way we do localisation in Nodes is a bit different than in other places. We
 Make sure to read our [**Swift Style Guide**](styleguide.md).  
 
 
-# Nodes CI
-The Nodes CI is our new continuous integration tool. Since it's still new, we don't really use it in many of our projects. However, all the new projects should have CI set up. [Here](https://engineering.nodesagency.com/articles/iOS/Continuous-Integration-on-iOS-with-HockeyApp-and-Testflight-Deployment/) you can find an article about how we got to build our own CI system. [Here](https://git.nodescloud.com/ios/ci-template) you can find a template for implementing our CI in your project.
+# Continuous Integration
+
+As a digital agency we build hundreds of apps during the day and to manage all these builds we are using Bitrise as a CI tool to speed up our development process and manage our client's certificates.
+
+Once you get access to Bitrise the creation process is straightforward and most of it is automatic if you have attached your github account to your Bitrise account. 
+
+[Here](./ci/bitrise-setup-readme.md) you will find a configuration guide to help you setting up your project on Bitrise.
```

Comment:
Can we add some headings above this and the below paragraphs?
Something like this:

1. Creating a new project on Bitrise

2. Integrating into your Xcode project

3. Troubleshooting